app/navigation/root-navigator.tsx

```diff
@@ -516,6 +519,23 @@ export const ChatNavigator = () => {
   )
 }
 
+const StackCards = createStackNavigator<CardStackParamList>()
+
+export const CardNavigator = () => {
```

Comment:
I think you don't need to make different navigation stack for Card screen. You can simply add CardScreen component to the Tab navigator like HomeScreen.

Reply:
ok I removed the extra navigation stack